components/Wizard/OCIAuthStep.js

```diff
@@ -0,0 +1,174 @@
+            <div className="pf-c-form__label pf-m-no-padding-top pf-l-flex pf-u-display-flex pf-m-justify-content-flex-start pf-m-nowrap">
+              <label htmlFor="user-input" className="pf-l-flex__item">
+                <span className="pf-c-form__label-text">
+                  <FormattedMessage defaultMessage="User OCID" />
+                </span>
+                <span className="pf-c-form__label-required" aria-hidden="true">
+                  &#42;
+                </span>
+              </label>
+              <Popover
+                id="user-popover"
+                bodyContent={
+                  <FormattedMessage
+                    defaultMessage="You can find your user OCID {iam} page in the OCI console."
+                    values={{
+                      iam: <strong>Identity and Access Management (IAM)</strong>,
+                    }}
+                  />
+                }
+                aria-label={formatMessage(ariaLabels.user)}
+              >
+                <Button variant="plain" aria-label={formatMessage(ariaLabels.user)}>
+                  <OutlinedQuestionCircleIcon id="popover-icon" />
+                </Button>
+              </Popover>
+            </div>
+            <TextInput
+              className="pf-c-form-control"
+              value={uploadSettings.user}
+              id="user-input"
+              name="user"
+              onChange={setUploadSettings}
+            />
+          </div>
+          <div className="pf-c-form__group">
+            <div className="pf-c-form__label pf-m-no-padding-top pf-l-flex pf-u-display-flex pf-m-justify-content-flex-start pf-m-nowrap">
+              <label htmlFor="private_key-input" className="pf-l-flex__item">
+                <span className="pf-c-form__label-text">
+                  <FormattedMessage defaultMessage="Private key" />
+                </span>
+                <span className="pf-c-form__label-required" aria-hidden="true">
+                  &#42;
+                </span>
+              </label>
+              <Popover
+                id="private_key-popover"
+                bodyContent={
+                  <FormattedMessage
+                    defaultMessage="You can view your deployed RSA keys in the API Keys section on the {iam} page in the OCI console."
+                    values={{
+                      iam: <strong>Identity and Access Management (IAM)</strong>,
+                    }}
+                  />
+                }
+                aria-label={formatMessage(ariaLabels.private_key)}
+              >
+                <Button variant="plain" aria-label={formatMessage(ariaLabels.private_key)}>
+                  <OutlinedQuestionCircleIcon id="popover-icon" />
+                </Button>
+              </Popover>
+            </div>
+            <FileUpload
+              id="private_key-input"
+              type="text"
+              name="private_key"
+              className="pf-c-form-control"
```

Comment:
Remove this line to prevent the double form input from showing.

Reply:
thanks! I was wondering why this was so ugly...